packages/agrid/agrid.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Adjustable grid (two dimensional array) library"
+description:
+  "Adjustable grids are two dimensional arrays whose width/height can be changed by adding or removing row/column at either end (one at a time)."
+maintainer: ["OCamlPro <contact@ocamlpro.com>"]
+authors: ["OCamlPro <contact@ocamlpro.com>"]
+license: "ISC"
+homepage: "https://github.com/ocamlpro/agrid"
+bug-reports: "https://github.com/ocamlpro/agrid/issues"
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "2.7" & >= "2.7"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```
That is a known bug in dune release, there is an open issue for that

Reply:
Ah indeed. Thanks !